services/brig/src/Brig/AWS.hs

```diff
@@ -121,9 +119,8 @@ mkEnv lgr opts emailOpts mgr = do
                      (mkEndpoint SQS.sqs      (Opt.sqsEndpoint opts))
                      (mkEndpoint DDB.dynamoDB (Opt.dynamoDBEndpoint opts))
     sq <- maybe (return Nothing) (fmap Just . getQueueUrl e . Opt.sesQueue) emailOpts
-    iq <- getQueueUrl e (Opt.internalQueue opts)
     jq <- maybe (return Nothing) (fmap Just . getQueueUrl e) (Opt.userJournalQueue opts)
-    return (Env g sq iq jq pk e)
```

Comment:
This is the conflict from the original PR:

```
<<<<<<< internal-queue
    return (Env g sq jq bl pk e)
=======
    return (Env g sq iq jq pk e)
>>>>>>> develop
```

I'm curious where has blacklist (`bl`) gone.

Reply:
https://github.com/wireapp/wire-server/pull/426 we now use C* for that.